src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rental = 0;
+  const d = days;
+
+  rental = d * 40;
+
+  if (d >= 3 && d < 7) {
+    rental -= 20;
+  }
+
+  if (d >= 7) {
+    rental -= 50;
+  }
```

Comment:
you should transfer all numbers to variables
for ex:
```
const  daysForBigDiscount = 7;
```
same for 3, 20 and 50
```suggestion
  if (days >= 7) {
    rental -= 50;
  }
  
  if (days >= 3) {
    rental -= 20;
  }
```